DigitalOffice/Services/CompanyService/Startup.cs

```diff
@@ -95,6 +79,39 @@ private void UpdateDatabase(IApplicationBuilder app)
             context.Database.Migrate();
         }
 
+        private void ConfigureRabbitMq(IServiceCollection services)
+        {
+            string appSettingSection = "ServiceInfo";
+            string serviceId = Configuration.GetSection(appSettingSection)["ID"];
+            string serviceName = Configuration.GetSection(appSettingSection)["Name"];
+
+            string appSettingMassTransitUris = "MassTransitUris";
+            string checkTokenUri = Configuration.GetSection(appSettingMassTransitUris)["CheckToken"];
+
```

Comment:
This is dumb. You already hard-code "MassTransitUris", so why the extra string variable?
Either come up with a different configuration or hard-code it there as well.